plugin/trino-ranger/src/main/java/io/trino/plugin/ranger/RangerSystemAccessControl.java

```diff
@@ -0,0 +1,1391 @@
+    }
+
+    @Override
+    public void checkCanCreateSchema(SystemSecurityContext context, CatalogSchemaName schema, Map<String, Object> properties)
+    {
+        if (!hasPermission(createResource(schema.getCatalogName(), schema.getSchemaName()), context, TrinoAccessType.CREATE, "CreateSchema")) {
+            AccessDeniedException.denyCreateSchema(schema.getSchemaName());
+        }
+    }
+
+    @Override
+    public void checkCanDropSchema(SystemSecurityContext context, CatalogSchemaName schema)
+    {
+        if (!hasPermission(createResource(schema.getCatalogName(), schema.getSchemaName()), context, TrinoAccessType.DROP, "DropSchema")) {
+            AccessDeniedException.denyDropSchema(schema.getSchemaName());
+        }
+    }
+
+    @Override
+    public void checkCanRenameSchema(SystemSecurityContext context, CatalogSchemaName schema, String newSchemaName)
+    {
+        if (!hasPermission(createResource(schema.getCatalogName(), schema.getSchemaName()), context, TrinoAccessType.ALTER, "RenameSchema")) {
+            AccessDeniedException.denyRenameSchema(schema.getSchemaName(), newSchemaName);
+        }
+    }
+
+    @Override
+    public void checkCanSetSchemaAuthorization(SystemSecurityContext context, CatalogSchemaName schema, TrinoPrincipal principal)
+    {
+        if (!hasPermission(createResource(schema.getCatalogName(), schema.getSchemaName()), context, TrinoAccessType.GRANT, "SetSchemaAuthorization")) {
+            AccessDeniedException.denySetSchemaAuthorization(schema.getSchemaName(), principal);
+        }
+    }
+
+    @Override
+    public void checkCanShowSchemas(SystemSecurityContext context, String catalogName)
+    {
+        if (!hasPermission(createResource(catalogName), context, TrinoAccessType.SHOW, "ShowSchemas")) {
+            AccessDeniedException.denyShowSchemas(catalogName);
+        }
+    }
+
+    @Override
+    public Set<String> filterSchemas(SystemSecurityContext context, String catalogName, Set<String> schemaNames)
+    {
+        Set<String> toExclude = null;
+
+        for (String schemaName : schemaNames) {
+            if (!hasPermissionForFilter(createResource(catalogName, schemaName), context, TrinoAccessType._ANY, "filterSchemas")) {
+                if (toExclude == null) {
+                    toExclude = new HashSet<>();
+                }
+
+                toExclude.add(schemaName);
+            }
+        }
+
+        return toExclude == null ? schemaNames : schemaNames.stream().filter(((Predicate<? super String>) toExclude::contains).negate()).collect(Collectors.toSet());
+    }
+
+    @Override
+    public void checkCanShowCreateSchema(SystemSecurityContext context, CatalogSchemaName schema)
+    {
+        if (!hasPermission(createResource(schema.getCatalogName(), schema.getSchemaName()), context, TrinoAccessType.SHOW, "ShowCreateSchema")) {
+            AccessDeniedException.denyShowCreateSchema(schema.getSchemaName());
```

Comment:
use static imports

Reply:
@kokosing  - there are 68 calls to static methods in `AccessDeniedException`, each are called only once. Replacing them to use static imports will result in 68 import statements (I guess use of * in import statements is not permitted). Please confirm if these calls should be replaced with static imports.